src/MSBuild/CommandLineSwitches.cs

```diff
@@ -488,6 +488,11 @@ internal void SetParameterlessSwitch(ParameterlessSwitch parameterlessSwitch, st
                 }
                 else
                 {
+                    if (IsMultipleAllowedSwithParameterDueToUnquote(unquoteParameters, parameterizedSwitch))
```

Comment:
Btw. what is the `Unquote` property of the `ParametrizedSwitchInfo` suppose to mean? Cannot we simply refer to just that here (plus the chnagewave)?

It's already set to `true` for targets: https://github.com/dotnet/msbuild/blob/5fcddc790f4eeaf953a3d283e39751dd0e1f2992/src/MSBuild/CommandLineSwitches.cs#L233

I'm wondering why the other swtich types should be handled differently.

Reply:
It is already being used by the method however, removing the validation on switch parameter to be equal to Target the check and unquoting will be applied to all unquote=True and multiple allowed parameters. This change was designed to reduce amount changed behaviours of different parameters, what do you think @JanKrivanek , @rainersigwald ?